core/src/main/scala/cats/Reducible.scala

```diff
@@ -60,6 +60,12 @@ import simulacrum.typeclass
   def reduceLeftTo[A, B](fa: F[A])(f: A => B)(g: (B, A) => B): B
 
   /**
+    * Monadic variant of `reduceLeftTo`
+    * */
```

Comment:
Minor nitpicks:

Could we change the last line of the scaladoc entry to just `*/`?

Also can we use `[[reduceLeftTo]]` so that the scaladoc has a handy link?